pkg/repositories/repositories.go

```diff
@@ -66,13 +97,13 @@ func SelectUnreleased(config *Config) {
 	// We use indexes since modifying objects while using them doesn't work in Golang as
 	// expected.
 	// More info: https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable
-	for categoryIdx := range config.Section.Categories {
-		for repoIdx, repo := range config.Section.Categories[categoryIdx].Repos {
+	for _, category := range config.Section.Categories {
```

Comment:
Since you're touching this code already, I think this is another method that should exist on the Config struct.